specification/communication/data-plane/Microsoft.CommunicationServicesIdentity/preview/2021-02-22-preview1/CommunicationIdentity.json

```diff
@@ -0,0 +1,313 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CommunicationIdentityClient",
+    "description": "Azure Communication Identity Service",
+    "version": "2021-02-22-preview1"
```

Comment:
Why it asses postfix preview1? 
Internally in code I can see this controller under V4 folder. 
As V2 has postfix preview2 .. then I suggest Use postfix preview4 ? Or what is your proposal?

Reply:
I need the version here to match the one in the TURN swagger to be able to generate the SDK correctly. TURN swagger has preview1 suffix as it's the first preview for TURN service endpoint. Hence I used preview1 suffix here.